day6/salestax.py

```diff
@@ -1,33 +1,68 @@
-import csv
-import pandas
 import fs 
+import csv
+import json
+
+import pandas as pd
+
+class calculation_of_tax:
+
+    def read_File(self):
+        with open('day6/input1.csv', 'r') as file:
+            reader = csv.reader(file, delimiter = '\t')
+            for row in reader:
+                print(row)
+
+    def csv_to_json(self, csvFilePath, jsonFilePath):
```

Comment:
keep the **variable name** and **argument name** in the snake case. so changes `csvFilePath` and `jsonFilePath` to `csv_file_path` and `json_file_path` respectively.